frontend/app-preview/src/components/AppBarConfig/AppPreviewBarConfig.tsx

```diff
@@ -1,36 +1,27 @@
 import React from 'react';
 import { RepositoryType } from 'app-shared/types/global';
-import { Link } from 'react-router-dom';
 import { TFunction } from 'i18next';
 import { editorPath } from 'app-shared/api/paths';
 import { Button, Select, LegacyToggleButtonGroup } from '@digdir/design-system-react';
 import { AltinnButtonActionItem } from 'app-shared/components/altinnHeader/types';
-import { TopBarMenu } from 'app-development/layout/AppBar/appBarConfig';
 import classes from '../AppPreviewSubMenu.module.css';
 import { ArrowCirclepathIcon, EyeIcon, LinkIcon } from '@navikt/aksel-icons';
 import { useTranslation } from 'react-i18next';
 import { AppPreviewSubMenuProps } from '../AppPreviewSubMenu';
 import { useLayoutSetsQuery } from '../../../../packages/ux-editor/src/hooks/queries/useLayoutSetsQuery';
 import { useStudioUrlParams } from 'app-shared/hooks/useStudioUrlParams';
+import { RoutePaths } from 'app-development/enums/RoutePaths';
```

Comment:
hm, this package should not import from app-development. 😊 

There are two ways to solve this issue. Alternative one is to make a new enum within the app-preview package with the needed routes. Then we as developers need to keep this in sync between thoose packages. 

Antoher option is to pass a "redirect url" as a prop to app-preview. This means that app-preview could be used by other packages and every package just pass a config about which url to navigate back to. This approach makes the app-preview package more standalone. 😊

Reply:
@framitdavid this is fixed now 😄